pipelines.adoc

```diff
@@ -94,7 +94,7 @@ You can use the link:https://github.com/tektoncd/experimental/blob/master/webhoo
 
 If you are developing a new pipeline and want to test it in a tight loop, you might want to use a script or manually drive the pipeline.
 
-. Log in to your cluster. For example for OKD,
+. Log in to your cluster. For example,
```

Comment:
see previous comment

Reply:
It’s fine as an example for now. Soon we’ll be back to OKD.